packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -90,11 +90,6 @@ class BottomSheet extends Component {
 			isMaxHeightSet: true,
 			isFullScreen: this.props.isFullScreen || false,
 		};
-
-		SafeArea.getSafeAreaInsetsForRootView().then(
-			this.onSafeAreaInsetsUpdate
-		);
-		Dimensions.addEventListener( 'change', this.onDimensionsChange );
```

Comment:
I spotted that these calls could trigger state changes before the component is mounted, for this reason, I decided to move it to `componentDidMount`.